From b786a5b008c22ecd1b6c213d5b07d2617c35d1d5 Mon Sep 17 00:00:00 2001
From: Miguel Horta <geral@miguelhorta.name>
Date: Wed, 10 Apr 2024 22:35:17 +0100
Subject: [PATCH 0905/1053] change: Keep tooltip location logic from upstream

It makes no sense, especialy because it is tied to have title bars
enabled
---
 src/misc_gui.cpp | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/src/misc_gui.cpp b/src/misc_gui.cpp
index bc3881267f..5660acd404 100644
--- a/src/misc_gui.cpp
+++ b/src/misc_gui.cpp
@@ -711,14 +711,6 @@ struct TooltipsWindow : public Window
 		if (pt.y + sm_height > scr_bot) pt.y = std::min(_cursor.pos.y + _cursor.total_offs.y - 5, scr_bot) - sm_height;
 		pt.x = sm_width >= _screen.width ? 0 : SoftClamp(_cursor.pos.x - (sm_width >> 1), 0, _screen.width - sm_width);
 
-		if (_settings_client.gui.windows_titlebars) {
-			// Move it to the top of the screen, away from mouse cursor, so it won't steal screen taps on Android
-			pt.y = GetMainViewTop();
-			if (_cursor.pos.y < pt.y + (int)GetMinButtonSize()) {
-				pt.x = _cursor.pos.x > _screen.width / 2 ? GetMinButtonSize() : _screen.width - sm_width - GetMinButtonSize();
-			}
-		}
-
 		return pt;
 	}
 
-- 
2.55.0

